pkg/components/state/pluggable.go

```diff
@@ -321,6 +322,18 @@ func (ss *grpcStateStore) Multi(ctx context.Context, request *state.Transactiona
 	return err
 }
 
+// MultiMaxSize returns the maximum number of operations allowed in a transactional request.
+func (ss *grpcStateStore) MultiMaxSize() int {
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	defer cancel()
+
+	resp, err := ss.Client.MultiMaxSize(ctx, new(proto.MultiMaxSizeRequest))
+	if err != nil {
```

Comment:
Should handle errors depending on the error.

If the error is `Unimplemented`, then -1 is a valid answer. Otherwise, should probably return an error

Reply:
We need to satisfy the MultiMaxSize interface though, which doesn't return an error.

Reply:
Ok how about:

- If error is `Unimplemented`, return -1
- If there's another error, log it as a warning

Reply:
And what do we return if there is another error? Still a -1?

Reply:
Ok, added a log for any error, but we continue to return -1

Reply:
@ItalyPaleAle  Can we consider this resolved? :)